source/game.c

```diff
@@ -2998,10 +2996,11 @@ static inline void game_playing_process_input_and_state(void)
 
         if (lerped_temp_score > 0)
         {
-            display_temp_score(fx2uint(lerped_temp_score));
+            // Set the score display first because it's more important
+            // in case there isn't enough time within the frame to display both
+            display_score(fx2uint(lerped_score));
 
-            // We actually don't need to erase this because the score only increases
-            display_score(fx2uint(lerped_score)); // Set the score display
+            display_temp_score(fx2uint(lerped_temp_score));
```

Comment:
I'm a little confused at the reason behind switching these around. Does it matter?